noble-migration/src/bin/upgrade.rs

```diff
@@ -30,6 +30,8 @@ const EXTRA_APT_SOURCE: &str = "EXTRA_APT_SOURCE";
 ///
 /// Each stage needs to be idempotent so that it can be run multiple times
 /// in case it errors/crashes.
+///
+/// Keep this in sync with the version in noble-migration/tasks/main.yml
```

Comment:
This is technically touching outside of the AW-only release, but it's only a comment so I figured it's okay.